core/app/models/spree/shipment.rb

```diff
@@ -35,6 +35,7 @@ class Shipment < Spree::Base
     scope :with_state, ->(*s) { where(state: s) }
     # sort by most recent shipped_at, falling back to created_at. add "id desc" to make specs that involve this scope more deterministic.
     scope :reverse_chronological, -> { order('coalesce(spree_shipments.shipped_at, spree_shipments.created_at) desc', id: :desc) }
+    scope :by_store, lambda { |store| joins(:order).merge(Spree::Order.by_store(store)) }
```

Comment:
this is so nitpicky but i can't help it... can we use the same `->(store) {` syntax as the other scopes, instead of `lambda { |store|`?